test

```diff
@@ -1256,21 +1262,30 @@ test_watch_staking_ledgers() {
     receipt_chain_hash=$(cat ./epoch_0_ledger.json | jq -r .staking_ledger.${pk}.receipt_chain_hash)
 
     assert '1' $token
+    assert '0' $epoch
     assert $pk $delegate
     assert $pk $public_key
     assert '1000' $balance
+    assert 'mainnet' $network
+    assert $epoch0 $ledger_hash
     assert '3NK2tkzqqK5spR2sZ7tujjqPksL45M3UUrcA4WhCkeiPtnugyE2x' $voting_for
     assert '2mzbV7WevxLuchs2dAMY4vQBS6XttnCUF8Hvks4XNBQ5qiSGGBQe' $receipt_chain_hash
 
-    # copy epoch 42 staking ledger to watched directory
-    cp $STAKING_LEDGERS/mainnet-42-jxYFH645cwMMMDmDe7KnvTuKJ5Ev8zZbWtA73fDFn7Jyh8p6SwH.json ./staking-ledgers
+    # curl epoch 42 staking ledger from MinaExplorer to watched directory
+    epoch42=jxYFH645cwMMMDmDe7KnvTuKJ5Ev8zZbWtA73fDFn7Jyh8p6SwH
+    curl https://storage.googleapis.com/mina-explorer-ledgers/${epoch42}.json -o ./mainnet-42-${epoch42}.json
+    cp ./mainnet-42-${epoch42}.json ./staking-ledgers/mainnet-42-${epoch42}.json
```

Comment:
Is this file going be continue to be available permanently or should we host it ourselves somewhere?

Reply:
We should eventually host it ourselves